llvm/lib/ObjectYAML/MachOEmitter.cpp

```diff
@@ -372,7 +372,7 @@ Error MachOWriter::writeSectionData(raw_ostream &OS) {
 
 // The implementation of makeRelocationInfo and makeScatteredRelocationInfo is
 // consistent with how libObject parses MachO binary files. For the reference
```

Comment:
```suggestion
// consistent with how libObject parses MachO binary files. For reference
```
If you don't mind, please fix this comment as highlighted as part of this PR, since you're already modifying the comment.